src/nccl_ofi_rdma.c

```diff
@@ -2248,13 +2243,6 @@ static int finish_connect(nccl_net_ofi_rdma_send_comm_t *s_comm)
 	s_comm->conn_resp_req->free(s_comm->conn_resp_req, false);
 	s_comm->conn_resp_req = NULL;
 
-	/* Since communicator can be used by a different thread,
-	 * established connection should be signalized last and there
-	 * should be a barrier after the communicator initialization
-	 * is finalized */
-	__sync_synchronize();
-	s_comm->connected = true;
```

Comment:
we don't need the `connected` field in the send comm anymore, right?

Reply:
removed